CONTRIBUTING.md

```diff
@@ -0,0 +1,91 @@
+# Contributing to `ptvsd` 
+
+[![Build Status](https://ptvsd.visualstudio.com/_apis/public/build/definitions/557bd35a-f98d-4c49-9bc9-c7d548f78e4d/1/badge)](https://ptvsd.visualstudio.com/ptvsd/ptvsd%20Team/_build/index?definitionId=1)
+[![Build Status](https://travis-ci.org/Microsoft/ptvsd.svg?branch=master)](https://travis-ci.org/Microsoft/ptvsd)
+[![GitHub](https://img.shields.io/badge/license-MIT-brightgreen.svg)](https://raw.githubusercontent.com/Microsoft/ptvsd/master/LICENSE)
+[![PyPI](https://img.shields.io/pypi/v/ptvsd.svg)](https://pypi.org/project/ptvsd/)
+
+
+## Contributing a pull request
+This project welcomes contributions and suggestions.  Most contributions require you to agree to a
+Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
+the rights to use your contribution. For details, visit https://cla.microsoft.com.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide
+a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions
+provided by the bot. You will only need to do this once across all repos using our CLA.
+
+### Prerequisites
+These packages are needed to to run `ptvsd` tests:
+* setuptools
+* coverage
+* requests
+* flask
+* django
+* pytest
+
+Install from pypi
+```console
+-m pip install -U setuptools coverage requests flask django pytest
+```
+
+### Linting
+We use `flake8` for linting. These are the settings we use with linting:
+```console
+ignore = W,E24,E121,E123,E125,E126,E221,E226,E266,E704,E265,E722,E501,E731,E306,E401,E302,E222
+exclude =
+    ptvsd/_vendored/pydevd,
+    ./.eggs,
+    ./versioneer.py
+```
+VSC Python settings for Linting:
+```json
+"python.linting.flake8Enabled": true,
+"python.linting.pylintEnabled": false,
+"python.linting.flake8Args": [
+    "--ignore", "E24,E121,E123,E125,E126,E221,E226,E266,E704,E265,E722,E501,E731,E306,E401,E302,E222",
```

Comment:
This shouldn't be necessary, since flake8 will pick those directly from the flake config in the repo (including when you run it from VSC).

Also, for the list of disabled stuff, I think it would be best to link directly to the file from this section, so that when we update the settings there, we don't have to remember to keep this bit in sync.

Reply:
Agreed will point to that file.